translations/strings.yaml

```diff
@@ -443,10 +443,81 @@ mobileOnboarding:
   alreadyASubscriber: Already a subscriber?
 
 inAppAuth:
+  emailInputPlaceholder: Enter email
   changeEmailLink: Change email
   signInButton: Sign in
   signInSubtitle: Continue to Mozilla VPN by signing in with your Firefox account
+  createPasswordLabel: Create password
   forgotPasswordLink: Forgot your password?
   passwordInputPlaceholder: Enter password
   passwordInputLabel: Password
-  invalidPasswordErrorMessage: Invalid password. Please try again
+  passwordHintCharacterLength: 
+    value: Must be at least 8 characters
+    comment: This is a hint for the user - they must provide a password with at least 8 characters 
+  passwordHintEmailAddressAsPassword: 
+    value: Must not be your email address
+    comment: This is a hint for the user - they must provide a password that does not match their email. 
+  passwordHintCommonPassword: 
+    value: Must not be a common password
+    comment: This is a hint for the user - password they provide must not be a common password. Ex. password.
+  productUpdateOptInDescription: 
+    value: I’d like to receive product update emails from Firefox.
+    comment: User has the option to opt-in to receive product update emails from Firefox. They are asked this when creating an account with us. 
+  createAccountButton: Create account
+  finishAccountCreationDescription: 
+    value: Finish creating your Firefox account to continue to Mozilla VPN
+    comment: User message prompting user to finish creating their account to use Mozilla VPN. 
+  emailVerificationDescription: 
+    value: Open your email and enter the verification code it that was sent.
+    comment: User has been sent a verification email containing a code. 
+  invalidPasswordErrorMessage: 
+    value: Invalid password. Please try again
+    comment: User has submitted an invalid password.
+  invalidCodeErrorMessage: 
+    value: Invalid code. Please try again
+    comment: User provided did not provide a correct security code.
+  invalidEmailErrorMessage: 
+    value: Invalid email. Please try again
+    comment: Generic message for when user submits incorrect email.
+  invalidEmailFormatErrorMessage: 
+    value: Email format not supported. Please try again
```

Comment:
What's the difference between this and invalid email? I would assume they're triggered by the same checks…

Reply:
Something like `Sign in with this email type is not currently supported` or `Email format type not supported. Please try again with a different email` might make more sense here and make the nuance between this error and 'Invalid email' clearer. I'll follow up with UX.